plugins/optimization-detective/class-od-url-metrics-group-collection.php

```diff
@@ -78,7 +83,7 @@ public function __construct( array $url_metrics, array $breakpoints, int $sample
 		sort( $breakpoints );
 		$breakpoints = array_values( array_unique( $breakpoints, SORT_NUMERIC ) );
 		foreach ( $breakpoints as $breakpoint ) {
-			if ( $breakpoint <= 1 || PHP_INT_MAX === $breakpoint ) {
+			if ( ! is_int( $breakpoint ) || $breakpoint < 1 || PHP_INT_MAX === $breakpoint ) {
```

Comment:
This was a minor logic error that likely never would have been encountered, since no one should attempt to have a breakpoint of 1px.